### ISSUE TITLE
Optical character recognition (OCR) of text in freeform fields

### ISSUE THREAD

**Issue by rholbert** [sdaps/sdaps#15]
I'd like to see an enhancement added to the SDAPS project that would allow optical character recognition (OCR) of the text in freeform fields.

The Gamera Project may be a good place to start:

http://gamera.informatik.hsnr.de/

**Comment by przemo** [comment]
Is there any light in a tunnel?

**Comment by benzea** [comment]
> Is there any light in a tunnel?

Not really. Unless there is a ready to use tool that could be integrated. It feels like it shouldn't be that hard these days with all the neural network advancements to build something basic at least ...

But well, it would need someone doing the work. I can help with integration, but not much more, I think.
